src/mapeo-manager.js

```diff
@@ -741,15 +738,24 @@ export class MapeoManager extends TypedEmitter {
   }
 
   /**
-   * @returns {{ deviceId: string } & Partial<import('./schema/client.js').DeviceInfoParam>}
+   * @returns {(
+   *   {
+   *     deviceId: string;
+   *     deviceType: DeviceInfoParam['deviceType']
+   *   } & Partial<DeviceInfoParam>
+   * )}
    */
   getDeviceInfo() {
     const row = this.#db
       .select()
       .from(localDeviceInfoTable)
       .where(eq(localDeviceInfoTable.deviceId, this.#deviceId))
       .get()
-    return { deviceId: this.#deviceId, ...row?.deviceInfo }
+    return {
+      deviceId: this.#deviceId,
+      deviceType: 'device_type_unspecified',
+      ...row?.deviceInfo,
```

Comment:
just checking - this is relying on the object spread being done at the end? i.e. the desired outcome is something like:

```js
deviceType: row?.deviceInfo.deviceType || 'device_type_unspecified',
```

Reply:
Yup, exactly.